documentation/developers.md

```diff
@@ -65,6 +65,42 @@ might be implemented to utilize the `compare-levels` function:
                                        :z "y"))}))
 ```
 
+### Implementing Alerts
+
+Alerts are returned by the  status function and allow you to expose human
+readable information about the state of your application. This makes it easy
+for status to be displayed and checked by unopinionated consumers. This works
+best if all conditions which may effect the availability of your service
+result in an alert.
+
+Examples:
+```clj
+{:severity :error
+ :message "Cannot connect to database"
+ :type :my-service/databas-unavailable
+ :details {}}
+
+{:severity :warning
+ :message "Repo may not be initialized"
+ :type :my-service/uninitilized-repo
+ :details { :repo "some-repo"} }
+```
+
+- `:severity` : The severity of the alert represents how consumers should support it.
```

Comment:
I do honestly prefer severity that is decoupled from availability. There are some potentially severe issues that have nothing to do with whether the service will be able to respond - for example, we might be able to detect a security issue. In the case of replicated services, having broken replication is extremely important to surface (relative importance depends on the app) but will not affect the local service's ability to function.